app/models/manageiq/providers/redhat/inventory/parser/infra_manager/full.rb

```diff
@@ -0,0 +1,587 @@
+module ManageIQ::Providers::Redhat::Inventory::Parser::InfraManager::Full
```

Comment:
Can we share the code? Code climate complains about having the same code.

Reply:
I may become our main refresh so maybe it would be good to make it a bit cleaner